data/charactermap.json

```diff
@@ -1,48 +1,56 @@
-{ "id":43,	"name":"Superwoman",	"world":"DC Comics" },
-{ "id":44,	"name":"Zane",			"world":"Ninjago" },
-{ "id":45,  "name":"Green Arrow", "world":"DC Comics" },
-{ "id":46,  "name":"Supergirl", "world":"DC Comics" }
+{ "id":1,	"name":"Batman",				"world":"DC Comics" },
+{ "id":2,	"name":"Gandalf",				"world":"Lord of the Rings" },
+{ "id":3,	"name":"Wyldstyle",				"world":"The LEGO Movie" },
+{ "id":4,	"name":"Aquaman",				"world":"DC Comics" },
+{ "id":5,	"name":"Bad Cop",				"world":"The LEGO Movie" },
+{ "id":6,	"name":"Bane",					"world":"DC Comics" },
+{ "id":7,	"name":"Bart",					"world":"The Simpsons" },
+{ "id":8,	"name":"Benny",					"world":"The LEGO Movie" },
+{ "id":9,	"name":"Chell",					"world":"Portal 2" },
+{ "id":10,	"name":"Cole",					"world":"Ninjago" },
+{ "id":11,	"name":"Cragger",				"world":"Legends of Chima" },
+{ "id":12,	"name":"Cyborg",				"world":"DC Comics" },
+{ "id":13,	"name":"Cyberman",				"world":"Doctor Who" },
+{ "id":14,	"name":"Doc Brown",				"world":"Back to the Future" },
+{ "id":15,	"name":"The Doctor",			"world":"Doctor Who" },
+{ "id":16,	"name":"Emmet",					"world":"The LEGO Movie" },
+{ "id":17,	"name":"Eris",					"world":"Legends of Chima" },
+{ "id":18,	"name":"Gimli",					"world":"Lord of the Rings" },
+{ "id":19,	"name":"Smeagol",				"world":"Lord of the Rings" },
+{ "id":20,	"name":"Harley Quinn",			"world":"DC Comics" },
+{ "id":21,	"name":"Homer",					"world":"The Simpsons" },
+{ "id":22,	"name":"Jay",					"world":"Ninjago" },
+{ "id":23,	"name":"Joker",					"world":"DC Comics" },
+{ "id":24,	"name":"Kai",					"world":"Ninjago" },
+{ "id":25,	"name":"ACU",					"world":"Jurrasic Park" },
+{ "id":26,	"name":"Gamer Kid",				"world":"Midway Arcade" },
+{ "id":27,	"name":"Krusty",				"world":"The Simpsons" },
+{ "id":28,	"name":"Laval",					"world":"Legends of Chima" },
+{ "id":29,	"name":"Legolas",				"world":"Lord of the Rings" },
+{ "id":30,	"name":"Lloyd",					"world":"Ninjago" },
+{ "id":31,	"name":"Marty Mcfly",			"world":"Back to the Future" },
+{ "id":32,	"name":"Nya",					"world":"Ninjago" },
+{ "id":33,	"name":"Owen",					"world":"Jurrasic Park" },
+{ "id":34,	"name":"Peter Venkman",			"world":"Ghostbusters" },
+{ "id":35,	"name":"Slimer",				"world":"Ghostbusters" },
+{ "id":36,	"name":"Scooby Doo",			"world":"Scooby-Doo" },
+{ "id":37,	"name":"SenseiWu",				"world":"Ninjago" },
+{ "id":38,	"name":"Shaggy",				"world":"Scooby-Doo" },
+{ "id":39,	"name":"Stay Puft",				"world":"Ghostbusters" },
+{ "id":40,	"name":"Superman",				"world":"DC Comics" },
+{ "id":41,	"name":"Unikitty",				"world":"The LEGO Movie" },
+{ "id":42,	"name":"Wicked Witch",			"world":"Wizard of OZ" },
+{ "id":43,	"name":"Superwoman",			"world":"DC Comics" },
+{ "id":44,	"name":"Zane",		 			"world":"Ninjago" },
+{ "id":45,	"name":"Green Arrow",			"world":"DC Comics" },
+{ "id":46,	"name":"Supergirl",				"world":"DC Comics" },
+{ "id":47,	"name":"Abby Yates",			"world":"New Ghostbusters" },
```

Comment:
Everything above this line was just a change to align with the new position of `"world"`